src/hu_stuff.c

```diff
@@ -1061,10 +1064,25 @@ static void HU_UpdateCrosshair(void)
     
     crosshair_target = linetarget;
 
-    if (hud_crosshair_target && crosshair_target
-        && !(crosshair_target->flags & MF_SHADOW))
+    if (crosshair_target && !(crosshair_target->flags & MF_SHADOW))
     {
-      crosshair.cr = colrngs[hud_crosshair_target_color];
+      if (hud_crosshair_health == crosshealth_target)
```

Comment:
With the current implementation, both crosshair coloring by player health and by target health can be used at the same time. However, I'm pretty sure it could get confusing, since you target's might have a health percentage that sets the crosshair color to the same color it was already using. What should I do about this?